pytorch_tabnet/tab_model.py

```diff
@@ -689,11 +688,17 @@ def construct_loaders(self, X_train, y_train, X_valid, y_valid, weights,
             Training and validation dataloaders
         -------
         """
+        if isinstance(weights, int):
+            if weights == 1:
+                raise ValueError("Please provide a list of weights for regression.")
```

Comment:
Is there any other case where weights can be an `int`?

Reply:
default value for weights is 0 which mean no balancing at all.
1 means balanced sampling but does not make sense for regression.
Then you can specify weight per class but it does not make sense for regression either

So for regression the only accepted values are 0 or per sample weights